source/feature/reduce-luns-from-sd.html.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
The file should be located in 'source/develop/release-management/features/storage'